src/providers/postgres/qgspostgresprovider.cpp

```diff
@@ -830,12 +831,12 @@ bool QgsPostgresProvider::loadFields()
     if ( fieldName == mGeometryColumn )
       continue;
 
-    int fldtyp = result.PQftype( i );
+    Oid fldtyp = result.PQftype( i );
     int fldMod = result.PQfmod( i );
     int fieldPrec = -1;
-    int tableoid = result.PQftable( i );
+    Oid tableoid = result.PQftable( i );
```

Comment:
in some part of the master PR you used unsigned int instead of Oid... what's the reason?